about/index.md

```diff
@@ -4,7 +4,11 @@ title: About STIX
 toc: about_toc.html
 ---
 
-Structured Threat Information Expression (STIX™) is a structured language to represent the full range of cyber threat information. The [STIX whitepaper](/getting-started/whitepaper) describes the motivation and architecture behind STIX, however at a high level the STIX language consists of 8 key constructs and the relationships between them:
+Structured Threat Information Expression (STIX™) is a structured language for describing cyber threat information so it can be shared, stored, and otherwise used in a consistent manner that facilitates automation.
```

Comment:
@sbarnum likes to say "analyzed" here, so maybe "shared, stored, and analyzed in a consistent manner"